src/app/game-state/character.ts

```diff
@@ -79,9 +80,19 @@ const INITIAL_AGE = 18 * 365;
 
 export class Character {
 
-  constructor(private logService: LogService,
-    private titlecasePipe: TitleCasePipe){
-  }
+  constructor(
+    private logService: LogService,
+    private titlecasePipe: TitleCasePipe,
+    public mainLoopService: MainLoopService
+    ){
+      mainLoopService.frameSubject.subscribe(() => {
```

Comment:
Seems like this could be a regular tick subscription and have the same effect since this new tick is on the same interval period.

Reply:
It's not though. The tick subscriber calls Tick, which passes the Subject.next repeatedly for every tick. This only broadcasts it once per frame. I was able to get the 100 years of a particularly bogged save down to 1 second from 1.4 with this change alone so I know there's an effect.